build.gradle

```
@@ -58,6 +58,8 @@ subprojects { subproject ->
 
 	repositories {
 		mavenCentral()
+		// to get r8
+		google()
```

Please order these repositories alphabetically. We keep repositories in sorted order, in part to emphasize that their order does not matter, and in part to make duplicates easier to spot.

```suggestion
		google() // to get r8
		mavenCentral()
```